charts/sampling/templates/_prometheus_remote_write.river.txt

```diff
@@ -0,0 +1,35 @@
+{{- define "deployment.prometheus.remote_write" -}}
+prometheus.remote_write "grafana_cloud_prometheus" {
+  // https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.remote_write/
+  endpoint {
+    url = env("GRAFANA_CLOUD_PROMETHEUS_URL")
+
+    basic_auth {
+      username = env("GRAFANA_CLOUD_PROMETHEUS_USERNAME")
+      password = env("GRAFANA_CLOUD_API_KEY")
+    }
+  }
+}
+
+{{ end }}
+
+{{- define "statefulset.prometheus.remote_write" -}}
+prometheus.remote_write "grafana_cloud_prometheus" {
+  // https://grafana.com/docs/agent/latest/flow/reference/components/prometheus.remote_write/
+  endpoint {
+    url = env("GRAFANA_CLOUD_PROMETHEUS_URL")
+
+    basic_auth {
+      username = env("GRAFANA_CLOUD_PROMETHEUS_USERNAME")
+      password = env("GRAFANA_CLOUD_API_KEY")
+    }
+    queue_config {
+      retry_on_http_429 = false
+    }
+  }
+  external_labels = {
+    "__metrics_gen_instance" = env("POD_UID"),
```

Comment:
I started to think that adding this label would not fully work because when metrics generation is done on the edge, `target_info` metric name is used instead of `traces_target_info`.

Using "__metrics_gen_instance" will not apply adaptive metrics aggregations to `target_info` metric, because default rule will apply to `traces_target_info` metric name: https://github.com/grafana/deployment_tools/blob/44243c57fcb391d8853061210fe7bbe611bf5742/ksonnet/lib/mimir-aggregations/default-aggregations.json#L127-L134

Reply:
@mar4uk you're correct, we'd also have to include a default rule for `target_info`.  As an alternative I can change this to `collector_id`, but we'd need default rules for all metrics with that label in this case.

Reply:
yep. I created the dedicated page about how to configure adaptive metrics https://github.com/grafana/app-o11y/pull/626/files#diff-4ce44250fd12c678b647e0a4d2d119842486f6124e90c08a0005afb61cab96d4, the rules are not default, they need to be configured by user. Not ideal, but that could be good for the beginning. Later we can move it to default rules

Reply:
Any reason not to just add these as default?

Reply:
not really :) Probably only align with the Agent team to use the same rules (currently they suggested using `agent_id` label, instead of `collector_id`) 
I can create PR to deployment_tools and cc agent team